airbyte-webapp/src/views/Connector/ServiceForm/components/Controls/Instruction.tsx

```diff
@@ -47,8 +50,18 @@ const Instruction: React.FC<IProps> = ({
   documentationUrl,
 }) => {
   const [isSideViewOpen, setIsSideViewOpen] = useToggle(false);
+  const config = useConfig();
   const { data: docs, isLoading } = useDocumentation(documentationUrl);
 
+  const removeBaseUrl = (url: { path: string }) => {
+    if (url.path.match(/^..\/..\/.*/)) {
```

Comment:
:question: Could you give an example of what URLs we're trying to rewrite here? This currently matches everything beginning with two arbitrary letters than slash then two more arbitrary letters? My feeling of gut is we're trying to rewrite specifically `../../` URLs here (in which case the dots would need to be escaped in the regex). Also if that's the case we could improve this by simply using `startsWith`, which is a bit more performant, and easier to read than `/^\.\.\/\.\.\/.*` :)

Reply:
Good catch! Fixed